examples/introductory/polymorphicEffects.wyv

```diff
@@ -0,0 +1,64 @@
+type Generic
```

Comment:
Instead of adding this example, could you please split it into several small tests that check each combination of type and effect parameters separately and create a new test suite under https://github.com/wyvernlang/wyvern/tree/master/tools/src/wyvern/tools/tests?